src/Calendar/components/TimeGrid/components/Event/Event.js

```diff
@@ -30,7 +30,7 @@ const Event = ({
         setDragHandleCenterHeight(height);
       }
     }
-  }, [dragHandleCenterHeight]);
+  });
```

Comment:
We need to at least have `[]` or move the code outside of a the `useEffect` here. Otherwise this effect will fire at every rerender, and might as well be inlined in the component instead

Reply:
feel free to add these back as long as you can confirm it didn't cause the bug. 👍  I just reverted everything